.changelog/3383.txt

```diff
@@ -0,0 +1,15 @@
+```release-note:feature
+breaking-change: users who previously had agent `enable_debug` set in `global.server.extraConfig`, should remove the extraConfig entry to avoid configuration conflicts.
+breaking-change: users who previously had agent `telemetry` stanza settings for `disable_hostname`, `enable_host_metrics`, `prefix_filter`, `dogstatsd_addr`, and/or `dogstatsd_tags`, should remove the extraConfig entry/entries to avoid configuration conflicts.
+helm: introduces `global.metrics.datadogIntegration` overrides to streamline consul-k8s datadog integration.
```

Comment:
**suggestion:** I am thinking that `global.metrics.datadog` is simpler and more than enough context for an entry.

Reply:
Updated override name to `global.metrics.datadog`